Documentation/Input/InputProviders.md

```diff
@@ -6,17 +6,17 @@ Input providers are registered in the **Registered Service Providers Profile**,
 
 These are the input providers available out of the box, together with their corresponding controllers:
 
-Input Provider | Controllers
---- | ---
-Input Simulation Service | Simulated Hand
-Mouse Device Manager | Mouse
-OpenVR Device Manager | Generic OpenVR, Vive Wand, Vive Knuckles, Oculus Touch, Oculus Remote, Windows Mixed Reality OpenVR
-Unity Joystick Manager | Generic Joystick
-Unity Touch Device Manager | Unity Touch Controller
-Windows Dictation Input Provider | *None* *
-Windows Mixed Reality Device Manager | WMR Articulated Hand, WMR Controller, WMR GGV Hand
-Windows Speech Input Provider | *None* *
+| Input Provider | Controllers |
+| --- | --- |
+| [`Input Simulation Service`](xref:Microsoft.MixedReality.Toolkit.Input.InputSimulationService) | Simulated Hand |
+| [`Mouse Device Manager`](xref:Microsoft.MixedReality.Toolkit.Input.UnityInput.MouseDeviceManager) | Mouse  |
+| [`OpenVR Device Manager`](xref:Microsoft.MixedReality.Toolkit.OpenVR.Input.OpenVRDeviceManager) | Generic OpenVR, Vive Wand, Vive Knuckles, Oculus Touch, Oculus Remote, Windows Mixed Reality OpenVR  |
+| [`Unity Joystick Manager`](xref:Microsoft.MixedReality.Toolkit.Input.UnityInput.UnityJoystickManager) | Generic Joystick  |
+| [`Unity Touch Device Manager`](xref:Microsoft.MixedReality.Toolkit.Input.UnityInput.UnityTouchDeviceManager) | Unity Touch Controller  |
+| [`Windows Dictation Input Provider`](xref:Microsoft.MixedReality.Toolkit.Windows.Input.WindowsDictationInputProvider) | *None*  |
+| [`Windows Mixed Reality Device Manager`](xref:Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityDeviceManager) | WMR Articulated Hand, WMR Controller, WMR GGV Hand |
+| [`Windows Speech Input Provider`](xref:Microsoft.MixedReality.Toolkit.Windows.Input.WindowsSpeechInputProvider) | *None* |
 
 Dictation and Speech providers don't create any controllers, they raise their own specialized input events directly.
 
-Custom input providers can be created implementing the [`IMixedRealityInputDeviceManager`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityInputDeviceManager) interface.
\ No newline at end of file
+Custom input providers can be created by implementing the [`IMixedRealityInputDeviceManager`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityInputDeviceManager) interface.
```

Comment:
The existing line is the ultimate teaser... nothing for you to change here, I just wonder if it would have been better if we deleted this. It doesn't give enough useful stuff for people to operate on.

Reply:
Indeed, I'd say leave it as it at least gives people a direction. There should be work to come back and elaborate here